.github/workflows/pythonpackage.yml

```diff
@@ -9,6 +9,14 @@ on:
       - '**.css'
       - '**.lock'
       - 'Makefile'
+  push:
+    paths:
+      - '.github/workflows/pythonpackage.yml'
+      - '**.py'
+      - '**.pyi'
+      - '**.css'
+      - '**.lock'
+      - 'Makefile'
```

Comment:
I wanted to run pipeline on each `git push` to investigate possible problems quickly. This way I can [run pipelines in my fork](https://github.com/pavelkraleu/textual-dev/actions) after each commit.
I tried to use [YAML anchors](https://github.com/pavelkraleu/textual-dev/commit/b405a2662704293330fc9d0142d71045d8d9581c) to prevent code duplication, but they are not [supported on Github](https://github.com/actions/runner/issues/1182) so they were removed in later commit.

Reply:
It seemed to me that [ListItem](https://textual.textualize.io/widgets/list_item/#__tabbed_1_2) and [ListView](https://textual.textualize.io/widgets/list_view/) are basically same examples. So I have skipped one of them.  
Am I missing some diference between those two?

Reply:
Yes, you are right. Sorry.
Maybe the button should say "ListView / ListItem" or something similar.